.env

```diff
@@ -0,0 +1,5 @@
+REACT_APP_IMAGE_URL="https://apps.perseids.org/lsj/android-chrome-512x512.png"
+REACT_APP_SITE_NAME="The Perseids Project"
+REACT_APP_TITLE="The Perseids Project"
+REACT_APP_DESCRIPTION="The Perseids Project aims to support access to scholarship in Classics for students and members of the public at all levels of competence. We provide a suite of tools that foster language acquisition, facilitate working with documents, and encourage research."
+REACT_APP_URL="https://www.perseids.org/"
```

Comment:
Let's make some changes to these variables to better fit the particular application.

* `REACT_APP_SITE_NAME`: keep this the same since `og:site_name` refers to the larger site a project is part of.
* `REACT_APP_TITLE`: we should change this to `LSJ Lexicon`
* `REACT_APP_DESCRIPTION`: let's change this to `Liddell-Scott-Jones Greek-English Lexicon`
* `REACT_APP_URL`: we should change this to `https://apps.perseids.org/lsj/`